README.md

```diff
@@ -0,0 +1,104 @@
+![CI](https://github.com/pulp-platform/snitch_cluster/actions/workflows/ci.yml/badge.svg)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+# Snitch Cluster
+
+This repository hosts the hardware and software for the Snitch cluster and its generator. Snitch is a high-efficiency compute cluster platform focused on floating-point workloads. It is developed as part of the PULP project, a joint effort between ETH Zurich and the University of Bologna.
+
+## Getting Started
+
+To get started, check out the [getting started guide](https://pulp-platform.github.io/snitch/ug/getting_started/).
```

Comment:
Change the link to the documentation for `snitch` to `snitch_cluster`. I am not sure the `ug/getting_started` path is still correct though.